scss/_side-navigation.scss

```diff
@@ -0,0 +1,238 @@
+        background-color: var(--#{$prefix}side-nav-item-active-border);
+      }
+    }
+
+    svg,
+    img {
+      width: var(--#{$prefix}side-nav-item-icon-size);
+      min-width: var(--#{$prefix}side-nav-item-icon-size);
+      height: var(--#{$prefix}side-nav-item-icon-size);
+      margin: calc(-1 * var(--#{$prefix}side-nav-item-padding-top)) subtract(var(--#{$prefix}side-nav-item-spacing), var(--#{$prefix}side-nav-border-width)) calc(-1 * var(--#{$prefix}side-nav-item-padding-bottom)) 0; /* stylelint-disable-line function-disallowed-list */
+    }
+  }
+}
+
+.side-nav-toggle {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: var(--#{$prefix}side-nav-min-width);
+  height: var(--#{$prefix}side-nav-item-min-height);
+  margin-left: auto;
+  background-color: transparent;
+  border: 0;
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    left: 0;
+    height: var(--#{$prefix}side-nav-item-min-height);
+    content: "";
+  }
+
+  &::after {
+    margin-right: 1px;
+    content: "";
+    border: $caret-width solid transparent;
+    border-right-color: var(--#{$prefix}side-nav-item-color);
+    border-left: 0;
+  }
+
+  &.collapsed::after {
+    margin-left: 3px;
+    transform: rotate(180deg);
+  }
+}
+
+.side-nav-content {
+  display: flex;
+  height: subtract(100%, var(--#{$prefix}side-nav-item-min-height));
+}
+
+.side-nav-scrollable {
+  height: subtract(100%, var(--#{$prefix}side-nav-item-min-height));
+  overflow: hidden auto;
+  scrollbar-width: thin;
+
+  &::-webkit-scrollbar {
+    width: 8px;
+    background-color: rgba($black, .1);
+  }
+
+  &::-webkit-scrollbar-thumb {
+    background-color: rgba(192, 192, 192, .5);
+
+    &:hover {
+      background-color: rgba(128, 128, 128, .6);
+    }
+  }
+}
+
+@each $breakpoint in map-keys($grid-breakpoints) {
+  $next: breakpoint-next($breakpoint, $grid-breakpoints);
+  $infix: breakpoint-infix($next, $grid-breakpoints);
+
+  .offcanvas#{$infix} {
+    @include media-breakpoint-down($next) {
+      .side-nav > * {
+        border: 0;
+      }
+
+      .side-nav-toggle {
+        display: none;
+      }
+
+      .side-nav-content {
+        height: 100%;
+      }
+    }
+  }
+}
+
+// Themes
+.side-nav-accordion {
+  > * {
+    width: subtract(var(--#{$prefix}side-nav-max-width), var(--#{$prefix}side-nav-border-width));
+  }
+}
+
+.side-nav-collapsible {
+  .collapse-horizontal {
+    margin-left: var(--#{$prefix}side-nav-min-width);
+
+    > * {
+      width: subtract(subtract(var(--#{$prefix}side-nav-max-width), var(--#{$prefix}side-nav-min-width)), var(--#{$prefix}side-nav-border-width));
+      height: 0;
+    }
+  }
+
+  .side-nav-content {
+    position: absolute;
+    flex-direction: column;
+    width: 100%;
+  }
+}
+
+.side-nav-content-with-drawer {
+  background: linear-gradient(to right, #{$gray-900} 0%, #{$gray-900} 54px, #{$white} 54px, #{$white} 100%); // Needed because is used out of the side-nav context
```

Comment:
Same linear gradient for RTL?

Reply:
Good question, haven't checked before but it seems fine 
![image](https://user-images.githubusercontent.com/91960143/223775764-c51e8128-ae4f-485f-922a-f77f1651c23f.png)

Reply:
It looks good so ok for me.